nautobot/dcim/models/cables.py

```diff
@@ -361,6 +361,8 @@ def from_origin(cls, origin):
         if origin is None or origin.cable is None:
             return None
 
+        from nautobot.circuits.models import CircuitTermination
```

Comment:
Is there a reason why this import is here? I would imagine to avoid a circular import issue? If so, can you please add a brief comment?

Reply:
I can add a brief comment. I was following another file that used import at the bottom instead of the top. I can change this to the top if you would like.